test/e2e_node/cpu_manager_test.go

```diff
@@ -283,7 +283,9 @@ func runGuPodTest(f *framework.Framework) {
 		cpu1 = cpuList[1]
 	} else if isMultiNUMA() {
 		cpuList = cpuset.MustParse(getCoreSiblingList(0)).ToSlice()
-		cpu1 = cpuList[1]
+		if len(cpuList) > 1 {
```

Comment:
it is inside the `isMultiNUMA` branch, meaning, there are more than 1 cpu, correct? Not following why it may be just 1.

Reply:
it calls `getCoreSiblingList(0)` to get all cores that placed under the same NUMA node as the CPU 0, so if the NUMA node has only one core, that unusual but possible(with fake NUMA for example), it will return the list with only one element.